src/static/css/header.less

```diff
@@ -51,6 +51,7 @@
 		.site-logo {
 			.grid_column(6);
 			text-align: right;
+			margin-bottom: unit( 25px / @base-font-size-px, em );;
```

Comment:
We could do some other adjustments to tighten up the height of the header and align the site name's baseline with the logo's baseline, but that doesn't need to happen in this PR.